src/app/notifications/components/SearchFailure/styles.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import styled, { css, keyframes } from 'styled-components/macro';
+import { PlainButton } from '../../../components/Button';
+import Times from '../../../components/Times';
+import {
+  bookBannerDesktopMiniHeight,
+  bookBannerMobileMiniHeight,
+  toolbarDesktopHeight,
+  toolbarMobileHeight,
+  toolbarMobileSearchWrapperHeight
+} from '../../../content/components/constants';
+import { disablePrint } from '../../../content/components/utils/disablePrint';
+import theme from '../../../theme';
+import { inlineDisplayBreak } from '../../theme';
+import { Header } from '../Card';
+
+export const clearErrorAfter = 5000;
+export const fadeOutDuration = 1000;
+
+const bannerBackground = '#F8E8EB';
+const errorBorderColor = '#E297A0';
+const closeIconColor = '#EDBFC5';
+const errorTextColor = '#C23834';
+
+const hoveredCloseIconColor = errorBorderColor;
+
+const fadeOut = keyframes`
+  from {
+    opacity: 1;
+  }
+
+  to {
+    opacity: 0;
+  }
+`;
+
+// tslint:disable-next-line:variable-name
+export const BannerBodyWrapper = styled.div`
+  width: 100%;
+  margin: 0;
+  height: 0;
+  z-index: ${theme.zIndex.contentNotifications - 1};
```

Comment:
Well i felt like it still should behave like a notification in relation to other elements on the page and it's z index only matters for the other notifications. I'm not really sure about adding it there since book banner does something similar.